content/how-to/market-data/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Market Data Examples
+weight: 10
```

Comment:
You should put different weights. The menu gets disordered

Reply:
Oh but everything is one page in a category.

Reply:
Moved the how-to section below web-api at least.